### PR TITLE
emoji proofreading

### PR DESCRIPTION
**Pull request by reaganhenke**
Some emoji in the chat aren't rendering on my browser, suggesting changes.

### CODE REVIEW

File: dialogue-constants.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const GENERALMOOD_CUTE = [
       "noclue: So cuuuuuuuute 💖",
       "bibop: #cute",
       "caty: this is so cute 😻",
-      "hes: Cute alert! 🚨🙈🐶🦝🐥🦦",
+      "hes: Cute alert! 🚨🙈🐶🦝🐥",
```
Comment (reaganhenke) [inline dialogue-constants.js:19]: Screen Shot 2022-02-05 at 11 22 36 AM
I'm just suggesting removing emojis that render like this, feel free to replace

File: dialogue-friendshipstars.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const STARSMOOD_SAD = [
     suspicion: ANYSUS,
     segments: [
       "caty: bittersweet ending...",
-      "coolio: 🥲",
+      "coolio: :(",
```
Comment (reaganhenke) [inline dialogue-friendshipstars.js]: Not sure what this was supposed to be, feel free to replace

File: dialogue-pizzaconundrum.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const PIZZAMOOD_OMG = [
     segments: [
       "coolio: 🍍🍕 🤮",
       "77gus77: pizza toppings a bid deal! 🧐",
-      "bibop: 🍍 or not 🍍 that is the ❔",
+      "bibop: 🍍 or not 🍍 that is the ❓",
```
Comment (reaganhenke) [inline dialogue-pizzaconundrum.js:28]: The white question mark didn't have enough contrast for me to see it
Screen Shot 2022-02-05 at 11 31 21 AM
 Suggesting replacing with the red one instead

File: dialogue-pizzaconundrum.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ const PIZZAMOOD_SAD = [
   {
     suspicion: ANYSUS,
     segments: [
-      "hes: #GiveMAriaAHug 🫂❤️",
+      "hes: #GiveMAriaAHug ❤️",
       "bibop: sad story... 😿",
       "thelibrarian: No pizza for the brave ☹️",
       "hes: Poor Maria 🌧️☹️☁️",
```
Comment (reaganhenke) [inline dialogue-pizzaconundrum.js]: Just checking, are these frowny faces intentional? This is how I see it:
Screen Shot 2022-02-05 at 11 32 36 AM

File: dialogue-friendshipstars.js (hunk 2 above)
Comment (LaChapeliere) [inline dialogue-friendshipstars.js]:
```suggestion
      "coolio: 😿",
```

File: dialogue-pizzaconundrum.js (hunk 4 above)
Comment (LaChapeliere) [reply dialogue-pizzaconundrum.js]: Yes, that's the page for it: https://emojipedia.org/frowning-face/

**Review by LaChapeliere — APPROVED** [review]
I've replaced one emoji (hoping it displays properly for you) but the rest of your modifications work for me!
I can't find a logic behind what emojis are a problem o0 None are super recent adds to the unicode or emoji standard (earliest is 2015, latest is 2020) and they are all from different batches and widespread o0 Weird weird weird

File: dialogue-pizzaconundrum.js (hunk 4 above)
Comment (reaganhenke) [reply dialogue-pizzaconundrum.js]: That page shows an emoji, but in the screenshot it's just an outline. Which is correct?

Comment (reaganhenke) [inline dialogue-pizzaconundrum.js]:
```suggestion
      "hes: Poor Maria 🌧️🙁☁️",
```
How does this look? Or keep it the same

Comment (LaChapeliere) [reply dialogue-pizzaconundrum.js]: It looks exactly the same to me, so if it looks better for you it's a win!

Comment (LaChapeliere) [reply dialogue-pizzaconundrum.js]: Oh wait, no, it's less frowny. Doesn't matter, let's keep the one that works for you

File: dialogue-pizzaconundrum.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ const PIZZAMOOD_SAD = [
   {
     suspicion: ANYSUS,
     segments: [
-      "hes: #GiveMAriaAHug 🫂❤️",
+      "hes: #GiveMAriaAHug ❤️",
       "bibop: sad story... 😿",
       "thelibrarian: No pizza for the brave ☹️",
```
Comment (reaganhenke) [inline dialogue-pizzaconundrum.js]:
```suggestion
      "thelibrarian: No pizza for the brave 🙁",
```

**Pull request merged**